packages/disputer/test/Disputer.js

```diff
@@ -243,12 +247,12 @@ contract("Disputer.js", function(accounts) {
             transports: [new SpyTransport({ level: "info" }, { spy: spy })]
           });
 
-          // Create a new instance of the ExpiringMultiPartyClient & GasEstimator to construct the disputer
-          empClient = new ExpiringMultiPartyClient(
+          // Create a new instance of the FinancialContractClient & GasEstimator to construct the disputer
+          financialContractClient = new FinancialContractClient(
```

Comment:
maybe we can start abbreviating `financialContract` to `fc` or `fincon` in the future. I liked the compactness of emp as a var name

Reply:
ye, it's a good point! `fc` seems pretty good actually. Would you like me to do that in this PR or should I do it separately? we would specifically search for the term`financialContract` with and replace with `fc`. would not effect imports or anything like that or any of the other cases like `Financial Contract` used in comments.

WDYT?

Reply:
if its easy for you to do in this one go for it. otherwise its not really that big of a deal

Reply:
for now I'm going to leave it as it is if you ok with that?